src/main/webapp/archives.html

```diff
@@ -883,7 +883,7 @@ <h2>Ready, efficient, and productive with Karaf in a minute !</h2>
         <p class="pt-2"><a class="btn btn-primary" href="documentation.html" role="button">Read Documentation &raquo;</a></p>
       </div>
       <p class="float-right"><a href="#">Back to top</a></p>
-      <p>&copy; 2018 <a href="https://www.apache.org">Apache Software Foundation</a> -
+      <p>&copy; 2019 <a href="https://www.apache.org">Apache Software Foundation</a> -
```

Comment:
I think it would be better to use a range, like `2018-2019`.

`2018` was the year we created the website. Now, the copyright covers 2018 AND 2019.

Reply:
Changes done.